app/services/marketing_site.rb

```diff
@@ -11,7 +11,6 @@ class UnknownArticleException < StandardError; end
     manage-your-account/personal-key
     trouble-signing-in/face-or-touch-unlock
     verify-your-identity/accepted-identification-documents
-    verify-your-identity/accepted-state-issued-identification
```

Comment:
did we consider adding a redirect in the help center to catch this? or was it never working in the first place

Reply:
[I had brought it up in the related Slack discussion](https://gsa-tts.slack.com/archives/C056RD1NEHW/p1721328909258589?thread_ts=1721328844.300159&cid=C056RD1NEHW) and I think we should, but I don't know that that was captured in this ticket. @jmax-gsa do you know if it's going to be addressed here, or is there a separate ticket for it?

Reply:
I don't know; I'll bring the question up at standup and either do it as part of this, or drop a pointer to the other ticket if there is one.

Reply:
Per conversation w/ May during standup, I'll add the redirect. See her comment on the ticket.

Reply:
Redirecet added in https://github.com/GSA-TTS/identity-site/pull/1305